src/components/mortgage-investment-compare/index.js

```diff
@@ -0,0 +1,220 @@
+// References:
+// https://www.bankrate.com/calculators/mortgages/new-house-calculator.aspx
+// https://www.nerdwallet.com/mortgages/mortgage-rates/30-year-fixed
+// https://michaelbluejay.com/house/15vs30.html
+
+import React from 'react'
+
+import { Button, Col, Row } from 'reactstrap'
+
+import InputCard from './input-card'
+import AmortizationTable from './amortization-table'
+import Summary from './summary'
+
+import _ from 'lodash'
+import { FV, PMT } from 'formulajs/lib/financial'
+import { getYearly } from '../../utils/timeSeriesResultsByOption'
```

Comment:
nit: What's the grouping here?

I typically group imports by 3rd-party first, then local files, using auto-sort for each group

Reply:
I don't have a strict preference for grouping, so happy to change it to whatever you prefer. The grouping here was basically 

1. styling
2. utility functions

Reply:
@hdavidzhu I've taken care of all suggestions in this PR. Please feel free to reorder these in your next PRs in whatever way you prefer. I really don't mind about the order as long as there is some logic to it.